vic/extensions/rout_rvic/src/rout_init.c

```diff
@@ -95,7 +95,7 @@ rout_init(void)
     get_nc_field_int(rout.param_filename,
                      "source_time_offset",
                      &i1start, &rout.rout_param.nSources, ivar);
-    for (i = 0; i < rout.rout_param.nSources; i++) {
+    for (i = 0; i < rout.rout_paparam_filenameram.nSources; i++) {
```

Comment:
hoooo, dit is een foutje :P ik zal m er de volgende keer uithalen. als het goed is verander ik niets in rout_rvic